### PR TITLE
Feature/student profile endpoints

### PR DESCRIPTION
**Pull request by nuzurie**

### CODE REVIEW

File: Student/repository/student_repository.go [hunk 1]
```diff
@@ -2,61 +2,117 @@ package repository
...
+)
+
 func (r *studentRepository) Create(ctx context.Context, id string, st *domain.Student) error {
-	r.m.Lock()
-	defer r.m.Unlock()
+	tx, err := r.db.Begin(ctx)
+	if err != nil {
+		return errors.NewInternalServerError(err.Error())
+	}
+	defer tx.Rollback(ctx)
```
Comment (car12lin12) [inline Student/repository/student_repository.go:39]: what does rollback do?

File: app/urls.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package app
+
+import (
+	"github.com/airbenders/profile/Student/delivery/http"
+	"github.com/gin-gonic/gin"
+)
+
+func mapURLs(h *http.StudentHandler, r *gin.Engine) {
```
Comment (car12lin12) [inline app/urls.go]: minor: are more URL mappings going in this class? If so then should we name this mapStudentURLs?

**Review by car12lin12 — APPROVED** [review]
Great job Zubair!! Your code works well too! I didn't have any major issues, just some clarification questions in comments

File: Student/repository/student_repository.go (hunk 1 above)
Comment (nuzurie) [reply Student/repository/student_repository.go:39]: [This](https://www.tutorialspoint.com/postgresql/postgresql_transactions.htm) is what I read to understand why we need begin, commit and rollback.

File: app/urls.go (hunk 2 above)
Comment (nuzurie) [reply app/urls.go]: Fixed, thank you!

Comment (nuzurie) [reply app/urls.go]: Please resolve and reapprove once you are satisfied.

**Review by car12lin12 — APPROVED** [review]

**Pull request merged**